infra/api/id_manager/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.8-slim-buster
+EXPOSE $ID_MANAGER_API_PORT
+
+WORKDIR /usr/src/api/id_manager
+COPY . /usr/src/api/id_manager
+
+RUN pip install --upgrade pip
+RUN pip install -r requirements.txt
+RUN pip install -r infra/api/requirements.txt
```

Comment:
please move "infra/api/requirements.txt" -> requirements-infra.txt

Reply:
why?
it's strictly related to APIs containers.
why install it in every container, as it's context-based?